### PR TITLE
Improve CSS for About section

### PR DESCRIPTION
**Pull request by lyu4321**
Fix for #27 

Contributors

Removed the bullet points from the Contributors list.

### CODE REVIEW

File: src/css/popup.css [hunk 1]
```diff
@@ -302,3 +302,7 @@ label.hidden {
 .separator {
     margin-bottom: 5px;
 }
+
+#contributors-list {
```
Comment (jure965) [inline src/css/popup.css:306]: Please make it a class based selector.

File: src/css/popup.css [hunk 2]
```diff
@@ -302,3 +302,7 @@ label.hidden {
...
+    list-style-type: none;
+}
```
Comment (jure965) [inline src/css/popup.css:308]: Please add an empty line at the end of file.

File: src/css/settings.css [hunk 3]
```diff
@@ -1,38 +1,308 @@
-button {
-    background-color: #0060df;
+html, body {
```
Comment (jure965) [inline src/css/settings.css:1]: I don't think this file needs changes.

Comment (lyu4321) [reply src/css/settings.css:1]: Oops sorry about this, I don't know why I pushed this change. I'll remove it right away.

**Comment by jure965** [comment]
Looks OK

**Pull request merged**